client/src/model/state.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Class representing the current state of the app.
```

Comment:
Could you elaborate on how this class is used. Does it represent a global state that gets passed down to (and updated by) components? Or is it local to a component?

Reply:
Done!